schedule/src/Models.elm

```diff
@@ -90,6 +90,8 @@ type alias EventInstance =
     , locationIcon : String
     , videoState : String
     , videoUrl : Maybe String
+    , slidesUrl : Maybe String
+    , slidesFilename : String
```

Comment:
Same as in the `eventInstanceDecoder` - we don't have slides on `EventInstance`s?